src/SDKs/Authorization/Authorization.Tests/Tests/Permissions.ScenarioTests.cs

```diff
@@ -52,7 +52,8 @@ public void GetResourceGroupPermissions()
         {
             using (MockContext context = MockContext.Start(this.GetType().FullName))
             {
-                string groupName = TestUtilities.GenerateName("csmrg");
+				// csmrg group creation fails when running with SPN auth method in Record mode,change the group to ResourceGroupName
```

Comment:
Earlier we couldnt run this because of permissions issue of the SPN,Now the SPN has been granted the ownder permission to the subscription.Will rerun the test case and update this